src/components/Sponsors/ThankYouSection.js

```diff
@@ -0,0 +1,193 @@
+        width:"1056px", 
+        height:"344px",
+        marginTop: 'calc(2vw - 150px)',
+        [theme.breakpoints.down('1300')]: {
+            marginTop: 'calc(2vw - 150px)'
+        },
+        [theme.breakpoints.down('1025')]: {
+            marginTop: 'calc(2vw - 150px)'
+        },
+        [theme.breakpoints.down('550')]: {
+            marginTop: 'calc(2vw - 100px)',
+        },
+        [theme.breakpoints.down('426')]: {
+            visibility: 'hidden'
+        },
+    },
+    mobileBlobStyle: {
+        visibility: 'hidden',
+        [theme.breakpoints.down('426')]: {
+            marginTop: 'calc(2vw - 180px)',
+            visibility: 'visible',
+            width: '100%',
+        },
+    },
+    headerText: {
+        fontFamily: 'Josefin Sans',
+        fontStyle: 'normal',
+        fontWeight: '600',
+        fontSize: '28px',
+        lineHeight: '44px',
+        display: 'flex',
+        alignItems: 'center',
+        color: '#000000',
+        [theme.breakpoints.down('426')]: {
+            fontSize: '20px',
+            lineHeight: '32px'
+        },
+    },
+    bodyText: {
+        fontFamily: 'poppins',
+        fontStyle: 'normal',
+        fontWeight: '400',
+        fontSize: '18px',
+        lineHeight: '10px',
+        display: 'flex',
+        align: 'center',
+        color: '#000000',
+        [theme.breakpoints.down('426')]: {
+            position: 'absolute',
+            fontSize: '14px',
+            lineHeight: '10px'
+        },
+    },
+    alignBodyText: {
+        textAlign: "center"
+    },
+    handsStyle: {
+        position: 'absolute',
+        right: 'calc(10vw + 50px)',
+        marginTop: 'calc(10vw + 150px)',
+        [theme.breakpoints.down('1300')]: {
+            marginTop: 'calc(20vw + 50px)',
+            right: 'calc(2vw + 10px)'
+        },
+        [theme.breakpoints.down('1025')]: {
+            marginTop: 'calc(25vw + 50px)',
+            marginRight: 'calc(8vw - 100px)'
+        },
+        [theme.breakpoints.down('769')]: {
+            marginTop: 'calc(40vw - 100px)',
+            marginRight: 'calc(20vw - 100px)'
+        },
+        [theme.breakpoints.down('426')]: {
+            marginTop: 'calc(40vw + 200px) ',
+            left: 'calc(20vw + 50px)',
+        },
+    },
+}));
+
+export default function ThankYou() {
+
+    const classes = useStyles();
+
+    return (
+
+        <Grid className={classes.container} alignItems="center" justify="center">
+
+            <Grid item xs={12} sm={6} lg={8} xl={12} className={classes.starsStyle}>
+                <SVGLoad image={Stars} alt="sponsorStars" />
+            </Grid>
+
+            <Grid item xs={5} sm={4} lg={8} xl={12} className={classes.handsStyle}>
+                <SVGLoad image={Hands} alt="sponsorStars" />
+            </Grid>
+
+            <Grid item xs={12} sm={12} lg={8} xl={10} className={classes.blobStyle}>
+
+                <svg viewBox="0 0 1056 344" fill="none" xmlns="http://www.w3.org/2000/svg">
+
```

Comment:
Can you import the svg and then render it in the return instead of having the entire path?